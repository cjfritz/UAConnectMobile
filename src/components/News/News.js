import React, { Component } from 'react'; 
import { WebView } from 'react-native';
import { Button, View } from 'native-base';

class News extends Component {
  render() {
    return (
      <WebView
        source={ { uri: 'https://news.uark.edu' } }
        style={ { marginTop: 20 } }
      />
    );
  }
}

export default News;
