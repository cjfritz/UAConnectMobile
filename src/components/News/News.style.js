import { StyleSheet } from 'react-native';

export default StyleSheet.create({
  backButton: {
    alignSelf: 'center',
    flex: 1,
    resizeMode: 'cover',
  },
  headerView: {
    backgroundColor: 'darkred',
    flexDirection: 'row',
    height: '12%',
    width: '100%',
  },
});
