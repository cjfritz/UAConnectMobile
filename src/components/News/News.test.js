import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import News from './News';

describe('<News />', () => {
  it('renders with default props', () => {
    const shallowRenderer = new ShallowRenderer();
    shallowRenderer.render(<News />);
    expect(shallowRenderer.getRenderOutput()).toMatchSnapshot();
  });
});
