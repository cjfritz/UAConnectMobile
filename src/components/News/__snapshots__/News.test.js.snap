// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<News /> renders with default props 1`] = `
<Styled(Container)>
  <Component
    style={
      Object {
        "backgroundColor": "darkred",
        "flexDirection": "row",
        "height": "12%",
        "width": "100%",
      }
    }
  >
    <Styled(Button)
      disabled={true}
      onPress={[Function]}
      style={
        Object {
          "alignSelf": "center",
          "flex": 1,
          "resizeMode": "cover",
        }
      }
      transparent={true}
    >
      <Styled(Icon)
        name="arrow-back"
      />
    </Styled(Button)>
  </Component>
  <WebView
    onNavigationStateChange={[Function]}
    originWhitelist={
      Array [
        "http://*",
        "https://*",
      ]
    }
    source={
      Object {
        "uri": "https://news.uark.edu",
      }
    }
  />
  <withNavigation(NavigationEvents)
    onWillBlur={[Function]}
  />
</Styled(Container)>
`;
